examples/ssr/src/main.js

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue';
+import App from './App.vue';
+import { createRouter } from './router';
+import { createInstantSearch } from '../../../src/instantsearch'; // TODO: move this to 'vue-instantsearch'
+import algoliasearch from 'algoliasearch/lite';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+Vue.config.productionTip = false;
+
+export async function createApp({
```

Comment:
We use `async / await` inside the client, but IIRC it's a bit complex to have it correctly setup. 

Do we really want to advocate for this syntax?

Reply:
This isn't my code; but rather the one from Vue CLI, so it seems fine for me? 👍